code/src/index.css

```diff
@@ -5,9 +5,142 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background-color: black;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+h2 {
+  color: white;
+}
+
+p {
+  color: white;
+}
```

Comment:
same rule => could be combined

Reply:
Thanks for very valid feedback Olof, makes for a much cleaner code! Changes are now done :)